src/services/query-options.js

```diff
@@ -216,6 +217,15 @@ class QueryOptions {
       this._order.push([associationName, fieldName, order]);
       this._neededFields.add(sortField);
     } else {
+      if (
+        isMSSQL(this._model.sequelize)
+        && this._sequelizeInclude?.length
```

Comment:
You have no guarantee that sequelizeInclude.length won't change between calling sort() and the .sequelizeOptions getter

Reply:
Yes and what is your suggestion ?

Reply:
Add a `_sequelizeOrder()` private getter on top of the file that does the replacement?
Is it possible?